Incidents management tool updates (#9034)
DataDog/documentation: content/en/monitors/incident_management.md

Incidents and outages can be managed in Datadog alongside the metrics, traces, logs, and other data you are collecting. You can view and filter incidents that are relevant to you and your team. 

In the Datadog paradigm, any of the following are appropriate situations for declaring an incident:

* An issue is or may be impacting customers or internal services.
* You do not know whether you should call an incident.

## Usage

Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing and resolved incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].

### Creating an incident

#### From a graph

You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 

{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph">}}

#### From the Clipboard

> Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the Clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the Clipboard and then click **Add to New Incident**. Everything on the Clipboard is added to the incident as a signal.

Author: ```suggestion
Use the beta Datadog Clipboard to gather multiple signals like monitors, graphs, security signals, logs, or any data in Datadog and to generate an incident. To add a graph to the clipboard, copy any graph, and then select Open Clipboard. Continuing adding all of the relevant graphs and monitors to the clipboard, select them, and then click Add to New Incident. Everything on the clipboard is added to the incident as a signal.

You can also add these signals to active incidents by scrolling down and selecting the incident ID.
```

#### From the Clipboard

Use the beta [Datadog Clipboard][6] to gather multiple signals like monitors, graphs, security signals, logs, or any data in Datadog and to generate an incident. To add a graph to the clipboard, copy any graph, and then select Open Clipboard. Continuing adding all of the relevant graphs and monitors to the clipboard, select them, and then click Add to New Incident. Everything on the clipboard is added to the incident as a signal.

You can also add these signals to active incidents by scrolling down and selecting the incident ID.

{{< img src="monitors/incidents/from-clipboard.png" alt="Add a dashboard to the clipboard">}}

{{< img src="monitors/incidents/clipboard.png" alt="Create in incident from the clipboard">}}

**Note**: Data on the Clipboard can also be exported to a new or existing dashboard or a notebook in addition to an incident.

#### From a monitor

You can declare an incident directly from a monitor by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the monitor is added into the incident as a signal.

> {{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a monitor">}}

Reviewer: <img width="487" alt="Screen Shot 2020-11-19 at 2 10 37 PM" src="https://user-images.githubusercontent.com/4602523/99712441-1102cf80-2a71-11eb-8210-77b8a64cf5f7.png">

These can also be added to existing incidents

{{< img src="monitors/incidents/incidents-top-1.png" alt="Incident Management"  style="width:80%;">}}

> Any event that may lead to a disruption in your organization’s services can be described as an incident, and it is often necessary to have a set framework for handling these events. Datadog’s Incident Management feature provides a system through which your organization can effectively identify and mitigate incidents. 

Reviewer: Datadog’s Incident Management product provides a system through which your organization can effectively identify and mitigate incidents.

> Incidents live in Datadog alongside the metrics, traces, and logs you are collecting. You can view and filter incidents that are relevant to you.

Reviewer: Incidents and outages can be managed in Datadog alongside the metrics, traces, logs, and other data you are collecting. You can view and filter incidents that are relevant to you and your team.

> * An issue is or may be impacting customers or services.

Reviewer: An issue is or may be impacting customers or internal services.

> * You do not know whether you should call an incident. Notify other people and increase severity appropriately.

Reviewer: Remove - Notify other people and increase severity appropriately.

## Usage

> Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing incidents.  You can configure additional fields that appear for all incidents in [Incident Settings][2].

Reviewer: all ongoing and resolved incidents.

> Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing incidents.  You can configure additional fields that appear for all incidents in [Incident Settings][2].

Reviewer 2: ```suggestion
Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].
```

### Creating an incident

#### From a graph

You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 

{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph">}}

> #### From the clipboard

Reviewer 2: ```suggestion
#### From the Clipboard
```

get it together

> Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the clipboard and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.

Reviewer: Use the Datadog Clipboard to gather multiple signals like monitors, graphs, security signals, logs, or any data in Datadog and to generate an incident. To add a graph to the clipboard, copy any graph, and then select **Open Clipboard**. Continuing adding all of the relevant graphs and monitors to the clipboard, select them, and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.

You can also add these signals to active incidents by scrolling down and selecting the incident ID. 

Note: the Datadog Clipboard is in beta

We should also link to docs

> Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the clipboard and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.

Reviewer 2: ```suggestion
Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the Clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the Clipboard and then click **Add to New Incident**. Everything on the Clipboard is added to the incident as a signal.
```

{{< img src="monitors/incidents/from-clipboard.png" alt="Add a dashboard to the clipboard">}}

{{< img src="monitors/incidents/clipboard.png" alt="Create in incident from the clipboard">}}

> **Note**: Data on the clipboard can also be exported to a new dashboard or a notebook in addition to an incident.

Reviewer 2: ```suggestion
**Note**: Data on the Clipboard can also be exported to a new dashboard or a notebook in addition to an incident.
```

Reviewer 3: Data on the Clipboard can also be exported to a new or existing dashboard or notebook in addition to an incident.

#### From a monitor

You can declare an incident directly from a monitor by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the monitor is added into the incident as a signal.

{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a monitor">}}

#### From the incidents page

In the [incidents UI][1], click the **New Incident** button to create an incident.

#### From Slack

Once you have the [Datadog integration enabled on Slack][3], you can use `/datadog` to declare a new incident from any channel.

{{< img src="monitors/incidents/from-slack.png" alt="Create in incident from Slack">}}

If the user declaring the incident is a part of your Datadog account, then that user becomes the Incident Commander (IC) by default. If the person declaring an incident is not part of your Datadog account, then the IC is assigned to a generic `Slack app user`. The IC can be changed on the [incidents page][1] in the Datadog app.

Once you declare an incident from Slack, it generates an incident channel.

> For more information about the Datadog Slack integration, check out [the docs][3].

Reviewer 2: ```suggestion
For more information about the Datadog Slack integration, see [the documentation][3].
```

{{< img src="monitors/incidents/workflow-1-graph-1.png" alt="From Graph"  style="width:80%;">}}

### 2. Declare an incident and assemble your team

Use the New Incident modal to assemble your team and notify them. The graph from which you created the incident is automatically attached as a signal. Attach any other signals that would give your team the necessary context to begin resolving this issue. The Slack and PagerDuty integrations enable you to send notifications through those services.

{{< img src="monitors/incidents/workflow-2-modal-1.png" alt="New Incident"  style="width:60%;">}}

### 3. Communicate and begin troubleshooting

If you have the [Datadog Slack app][3] installed, the Slack integration can automatically create a new channel dedicated to the incident, so you can consolidate communication with your team and begin troubleshooting.

For non-EU customers who use Slack, [sign up for beta access][4] to the Datadog Slack app. For EU customers who use Slack, stay informed about the Slack app by emailing support@datadoghq.com.

> {{< img src="monitors/incidents/workflow-3-slack-1.png" alt="Communicate"  style="width:80%;">}}

Reviewer: <img width="884" alt="Screen Shot 2020-11-05 at 9 07 40 PM" src="https://user-images.githubusercontent.com/4602523/99714139-385a9c00-2a73-11eb-9e4d-0b3361c7d38c.png">

Reviewer: image replacement
